docs/docsite/rst/galaxy.rst

```diff
@@ -134,6 +134,10 @@ Use the following example as a guide for specifying roles in *requirements.yml*:
       scm: git
       version: "0.1"  # quoted, so YAML doesn't parse this as a floating-point value
 
+    # using git+ssh:// URI scheme
+    - src: git+ssh://git@github.com:vmindru/ansible.git
```

Comment:
This format is no different that the one above referenced as `from GitLab or other git-based scm` except that it adds the scheme which is redundant.

The `git+ssh://` scheme is inferred when using an scp-like syntax.

Reply:
hi, tahnks for the comment. can you be more specific about "scp-like" syntax?

Reply:
Please see https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_the_ssh_protocol